ibis/backends/base/sql/alchemy/registry.py

```diff
@@ -506,6 +506,9 @@ def _string_join(t, expr):
     # other
     ops.SortKey: _sort_key,
     ops.Date: unary(lambda arg: sa.cast(arg, sa.DATE)),
+    ops.DateFromYMD: fixed_arity(sa.func.date, 3),
+    ops.TimeFromHMS: varargs(sa.func.time),
+    ops.TimestampFromYMDHMS: varargs(sa.func.make_timestamp),
```

Comment:
Are these functions actually variadic?

Reply:
We should probably be more explicit in our developer guide here, but generally I think we want to avoid implicit behavior like that and even though it's more annoying to write out the explicit translation rule, it's also less error prone and has no implicit behavior.

That said, this PR has been open for a while and I'm okay with this bit as is.

Reply:
Sorry, I just realized that 

> I'm okay with this bit as is

conflicts with what I wrote above. Would you mind implementing the rule directly? That's should be the final thing and then I'll merge it!